addons/event/static/src/client_action/event_registration_summary_dialog.js

```diff
@@ -182,7 +182,11 @@ export class EventRegistrationSummaryDialog extends Component {
         const [{ id: reportId }] = await this.orm.searchRead("ir.actions.report", [["report_name", "=", reportName]], ["id"]);
 
         this.notification.add(
-            _t("'%(name)s' badge sent to printer '%(printer)s'", { name: this.registration.name, printer: this.selectedPrinter.name }),
+            _t("'%(name)s' %(type)s badge sent to printer '%(printer)s'", {
+                name: this.registration.name,
+                type: this.registration.ticket_name,
```

Comment:
Ticket type isn't required, so `ticket_name` could be `false`, we should check it first before including it in the message